tests/app/test_jinja_filters.py

```diff
@@ -1339,5 +1340,50 @@ def test_map_list_collector_config_render_icon_set():
     assert output == expected
 
 
+def test_map_list_config():
+    list_values = [
```

Comment:
Minor but this is the wrong type at the moment, no need for the nested list, supposed to be a list of dicts.